meson_test.py

```diff
@@ -144,6 +146,11 @@ def drain_futures(futures):
         (result, numlen, tests, name, i, logfile, jsonlogfile) = i
         print_stats(numlen, tests, name, result.result(), i, logfile, jsonlogfile)
 
+def filter_tests(suite, tests):
+    if suite is None:
+        return tests
```

Comment:
This should consolidate tests that are the same but have different 'suite' names